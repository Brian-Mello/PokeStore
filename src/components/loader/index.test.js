import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '.';

describe('testando integridade do Loader component', () => {
    it(' Loader snapshot', () => {
        const loader = renderer.create(<Loader/>).toJSON();
        expect(loader).toMatchSnapshot();
    });
}); 