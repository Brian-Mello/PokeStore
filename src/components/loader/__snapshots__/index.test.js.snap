// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testando integridade do Loader component  Loader snapshot 1`] = `
<div
  className="sc-AxiKw guiDHr"
>
  <svg
    height="150"
    viewBox="0 0 40 60"
    width="150"
  >
    <polygon
      className="sc-AxirZ kMUHWQ"
      fill="none"
      points="16,1 32,32 1,32"
      stroke="black"
    />
    <text
      className="sc-AxjAm lcmTQl"
      fill="black"
      x="4"
      y="45"
    >
      Loading
    </text>
  </svg>
</div>
`;
